sway/tree/layout.c

```diff
@@ -100,14 +101,16 @@ void container_add_child(struct sway_container *parent,
 			parent, parent->type, parent->width, parent->height);
 	list_add(parent->children, child);
 	child->parent = parent;
+	// TODO: set focus for this container?
+	sway_input_manager_set_focus(input_manager, child);
```

Comment:
Will this always be correct? Example: start a process in a different workspace, pid tracking picks it up, switch away, window opens off-screen

Probably best to only set focus if the parent has focus?